pkg/reconciler/revision/resources/deploy_test.go

```diff
@@ -217,6 +219,13 @@ var (
 				},
 			},
 			ProgressDeadlineSeconds: ptr.Int32(0),
+			Strategy: appsv1.DeploymentStrategy{
+				Type: appsv1.RollingUpdateDeploymentStrategyType,
+				RollingUpdate: &appsv1.RollingUpdateDeployment{
```

Comment:
I actually discussed that with Matt some 2 years ago, though, I actually wanted to use more aggressive settings :)

Reply:
More aggressive in which ways?